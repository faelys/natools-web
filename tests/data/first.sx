(elements
   (html-body "\
<h1>First Page</h1>
<p>Data in the first page.</p>\n"
(render page-comments))
   (title "First Page"))
(tags (1 pages/news tags/topic1/subtopic))
(comments (backend comments 1) (tags comments) (filter comment-filter-bad))
