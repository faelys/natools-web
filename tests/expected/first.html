<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
  <head>
    <title>First Page</title>
  </head>
  <body>
<h1>First Page</h1>
<p>Data in the first page.</p>
    <div class="footer">
<p>Default footer</p>
    </div>
  </body>
</html>
